tests/reg_tests/test_MPhysGeo.py

```diff
@@ -0,0 +1,566 @@
+# Standard Python modules
+import copy
+import os
+import unittest
+
+# External modules
+from baseclasses.utils import Error
+import commonUtils
+import numpy as np
+from parameterized import parameterized_class
+from pyspline import Curve
+from stl import mesh
+
+# First party modules
+from pygeo.mphys import OM_DVGEOCOMP
+
+try:
+    # External modules
+    from openmdao.api import Group, IndepVarComp, Problem
+    from openmdao.utils.assert_utils import assert_check_totals, assert_near_equal
+
+    omInstalled = True
+
+except ImportError:
+    omInstalled = False
+
+try:
+    # External modules
+    import pyOCSM  # noqa: F401
+
+    ocsmInstalled = True
+
+except ImportError:
+    ocsmInstalled = False
+
+
+# input files for all DVGeo types
+input_path = os.path.dirname(os.path.abspath(__file__))
+outerFFD = os.path.join(input_path, "../../input_files/outerBoxFFD.xyz")
```

Comment:
Should change `"../../input_files/etc"` to `os.path.join(input_path, "..", "..", "input_files", etc.)`

The whole point of using the join function is to eliminate the `/` character.

Going even further, I like `pathlib` much better, but that's extra credit.